
Ati Xpress memory detection fix from X.org upstream CVS.

  - Rene Rebe <rene@exactcode.de>

diff -u -r1.84 -r1.85
--- ./src/radeon_driver.c	2006/01/13 01:02:01	1.84
+++ ./src/radeon_driver.c	2006/01/19 14:20:37	1.85
@@ -2582,7 +2582,7 @@
     case PCI_CHIP_RS480_5954:
     case PCI_CHIP_RS482_5974:
 	info->ChipFamily = CHIP_FAMILY_RV380; /*CHIP_FAMILY_RS400*/
-	/*info->IsIGP = TRUE;*/ /* ??? */
+	info->IsIGP = TRUE;
 	/*info->HasSingleDAC = TRUE;*/ /* ??? */
         break;
 
